submissions/pocarski-combinators/pocarski-combinators.md

```diff
@@ -0,0 +1,47 @@
+
+### Basic examples
+
+Now, let's explore some cases where each of these modules might come in handy.
+
+Say you have a nuclear reactor blueprint where the extraction of a used fuel cell triggers the insertion of a fresh one. Such a design would have to be manually started since reactors are built empty. What you ideally want to add is a circuit which, once all fuel cell chests have items in them, triggers the fueling inserters exactly once. This is where the monostable comes in. Have a combinator in each chest checking if there are enough items in it, and then wire all of those together into a single combinator that checks if all chests are ready. This decider then outputs a "used fuel cell" signal into a monostable, which is wired to every fueling inserter of the reactor. This causes all fueling inserters to trigger exactly once the moment there is fuel available to all of them, starting the reactor automatically. By extension, this also makes the reactor automatically restart if it ever runs out of fuel.
+
+(reactor fueling)
+
+Next, a classic example: backup power. Imagine you have an array of accumulators and you want to activate your boilers if the stored energy gets too low. You could just wire a switch directly to an accumulator and tell it to activate if accumulators are below, say, 20% charge, but that would just cause it to rapidly switch on and off, keeping the accumulators at exactly 20% all the time. Instead, you should use an RS latch. Have a combinator output "S" when accumulator charge is below 20%, and another one output "R" when charge is above 70%. Hook them both to the latch, and wire the output of the latch to a switch set to activate if S > 0. The switch will activate as soon as charge drops below 20%, and keep the backup running until charge goes above 70%.
+
+(backup switch)
+
+Finally, a process that many fear to set up: uranium enrichment. The part we're worried about is the recycling of uranium. We want to stop recycling U-235 as soon as we hit 40 items, after which the extra one should be placed into the output. Naturally, we use a counter for this. The recycling inserter is set to "read hand - pulse", so every time it grabs uranium for recycling the counter counts up. As soon as the amount in the counter becomes greater or equal to 40, the recycling inserter's filter is changed to U-238. U-235 has higher signal priority, so this can be done by simply turning off "U-235" while there is a constant input of "U-238". This means it takes out U-238 on its fifth swing, which resets the counter and triggers the U-235 output inserter. We don't need any special handling of the recycled U-238 because the input inserter's stack size is 3, meaning there will never be any competition between U-238 inserters, since their respective item counts add up to 5, the exact amount needed.
```

Comment:
It may be better when accompanied by an annotated image, but this last sentence seems too long.